### DISCUSSION TITLE
Transfer of Ownership

### DISCUSSION THREAD

**Discussion by wdhdev** [is-a-dev/discussions#1] [General]
@phenax, If possible, I am requesting a transfer of ownership for the is-a.dev domain name on Namecheap.

I have been successfully maintaining the Cloudflare zone with few issues. However, I have one main issue I am unable to resolve without ownership of the domain. I am unable to add IPv6 glue records for the custom nameservers feature on Cloudflare to the domain name, as I do not own it. The only method of adding these is through Namecheap support, which do not allow domain managers to add these records, only domain owners. If I could get domain ownership, it would help me solve these issues, as well as any future ones.

I am willing to continue to run and ensure the domain name will stay renewed in the future, and am willing to take full ownership of the service.

I have sent you my Namecheap email address on Discord.

(I haven't been able to reach you via email so I have created this discussion instead.)

**Comment by phenax** [comment]
@wdhdev Done! Let me know once confirmed.

**Reply by wdhdev** [reply]
Thanks! I've accepted.

Thank you for running the service this far, I'll do my best to continue what you've left :)

**Reply by wdhdev** [reply]
@phenax Quick question, do you want to stay on the is-a.dev team as it is now, or do you want to fully step back?

**Reply by phenax** [reply]
I've been checked out of this for a while now so I'm out more officially. You guys have done a really good job with the project so I already know it's in good hands. Removed myself from the org on github. Farewell!

**Reply by wdhdev** [reply]
Farewell Akshay! Thank you for your service, I wish you the best in future :)
